<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DMSortingDialog</class>
 <widget class="QDialog" name="DMSortingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="datasetCB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>31</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>No dataset selected...</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="selectButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>171</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listLeftVarWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>101</y>
     <width>131</width>
     <height>167</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listRightVarWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>101</y>
     <width>141</width>
     <height>167</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>71</y>
     <width>213</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Variables:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="orderListWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>101</y>
     <width>241</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxParallel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="serNparRB">
       <property name="text">
        <string>Serial n Parallel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="parRB">
       <property name="text">
        <string>Parallel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="DatasetLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Dataset :</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="loadDECheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Data Editor after sorting</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DMSortingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DMSortingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
